dev-python/h11/h11-0.8.1.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="A pure-Python implementation of HTTP/1.1 inspired by hyper-h2"
+HOMEPAGE="https://pypi.python.org/pypi/h11"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Please include running tests. If tests require live network access, also add `RESTRICT="test"` with appropriate comment.

https://wiki.gentoo.org/wiki/Project:Python/Tests

Reply:
Tests successfully pass on both ebuilds. I tried first to run them via unittest, but it wouldn't discover a single test. I invested some time into figuring out how to make it work, but eventually gave up and had py.test do the testing (which worked out like a charm from the 1st try).

The tests do not require internet access. I was able to emerge h11 package on a laptop entirely disconnected from the internet and tests passed too.

Reply:
They actually `import pytest`, so py.test is the way to go.